.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "regfile"]
+	path = regfile
+	url = git@github.com:hdavidzhu/HW4.git
```

Comment:
@srli Here is my submodule. You can check out the file I'm referencing. It's just a folder in this repository.